cluster/saltbase/salt/top.sls

```diff
@@ -35,7 +39,11 @@ base:
     - kube-apiserver
     - kube-controller-manager
     - kube-scheduler
+{% if grains['cloud'] is defined and grains.cloud == 'gce' %}
```

Comment:
eventually, as above, doing this piecemeal so that I don't break things I haven't tested.